src/main/java/com/lothrazar/cyclic/item/AntimatterEvaporatorWandItem.java

```diff
@@ -0,0 +1,130 @@
+package com.lothrazar.cyclic.item;
+
+import com.lothrazar.cyclic.base.ItemBase;
+import com.lothrazar.cyclic.registry.SoundRegistry;
+import com.lothrazar.cyclic.util.UtilShape;
+import com.lothrazar.cyclic.util.UtilSound;
+import net.minecraft.block.Blocks;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.Fluids;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemUseContext;
+import net.minecraft.util.*;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraft.world.World;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class AntimatterEvaporatorWandItem extends ItemBase {
+
+  public static final int COOLDOWN = 15;
+  private static final List<Fluid> VALID_WATER = new ArrayList<>();
+  private static final List<Fluid> VALID_LAVA = new ArrayList<>();
+
+  public enum Mode implements IStringSerializable {
+    WATER, LAVA, GENERIC;
+
+    @Override
+    public String getString() {
+      switch (this) {
```

Comment:
also you can do something like 

   return this.name().toLowerCase();
   
   since name() is already built into Enum.class but its uppercase